package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cielo-finance/component-library",
-  "version": "1.0.41",
+  "version": "1.0.42",
```

Comment:
@shan8851 when i do toggle dark/light mode , logo stay without changes 
![Screenshot 2022-11-22 at 19 18 55](https://user-images.githubusercontent.com/35912161/203379661-0ea982b4-f1b0-42dd-b4d4-6a33b0af655b.png)

Reply:
If you switch it in actions you can see it working. 
![Screenshot 2022-11-23 at 14 15 53](https://user-images.githubusercontent.com/39712238/203569029-ecb04cda-1ccb-44a6-87e2-11159faf4f33.png)

I wasn't sure how to use theme in storybook, but fixed it so it works now